backend/db/migrations/20171116292217-create-entry.js

```diff
@@ -0,0 +1,79 @@
+export function up (queryInterface, Sequelize) {
+  return queryInterface.createTable('entries', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    showId: {
+      type: Sequelize.INTEGER,
+      references: {
+        model: 'shows',
+        key: 'id'
+      },
+      onUpdate: 'cascade',
+      onDelete: 'cascade'
+    },
+    studentUsername: {
```

Comment:
I think this follows your erd diagram, not Bill's. Did we decide to use your approach instead? Or was it just much harder to implement his his method?

Reply:
Unfortunately `entityId` can't be both type INT (to support groups) and STRING (to support users / usernames).